posts/2020-03-04-Rust-survey-2019.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Rust Survey 2019 Results"
+author: The Rust Survey Team
+---
+
+Greetings Rustaceans!
+
+We are happy to present the results of our fourth annual survey. The survey was available in **14** different languages and we received **3997** responses. 
+
+Here is the language distribution of the responses we received.
+
+* English: 69.6%
+* Chinese: 10.8%
+* German: 4.3%
+* French: 3.3%
+* Japanese: 3.0%
+* Polish: 1.2%
+* Portuguese: 1.2%
+* Spanish: .9%
+* Korean: .8%
+* Italian: .6%
+* Swedish: .5%
+* Vietnamese: .2%
+
+![Rust Survey Language Distribution](/images/2020-03-RustSurvey/1-Language-Breakdown.png)
+
+![Do You Use Rust](/images/2020-03-RustSurvey/6-Have-You-Used-Rust.png)
+
+About 75% of users indicated they were actively using Rust.
+
+# Rust Users
+
+First, let's go over the results for Rust users.
+
+## **Time with Rust**
+
+![How long have you worked in Rust](/images/2020-03-RustSurvey/7-How-Long-Worked-With-Rust.png)
+
+We asked our users how long they had been working with Rust. The majority had been using Rust for 1-2 years. Overall these numbers were not significantly different from what we saw in 2018.
+
+![How long did it take to be productive](/images/2020-03-RustSurvey/8-How-Long-To-Be-Productive.png)
+
+About 37% of Rust users felt productive in Rust in less than a month of use - this is not too different from the percentage last year (40%). Over 70% felt productive in their first year. Unfortunately, like last year, there is still a struggle among users - 21% indicated they did not yet feel productive.
+
+![How long have you been unproductive](/images/2020-03-RustSurvey/9-How-Long-Unproductive.png)
+
+Additionally, many users indicated that they were not feeling comfortable in Rust - even after over 3 months of use. We need to find ways to help bridge users to productivity so they don't get stuck.
```

Comment:
I think there is more analysis we could do here. For example, what do people *mean* by unproductive? This might be an issue with language complexity, but it might also relate to tooling or other obstacles (IDEs, in particular). I would mention some of the possibilities and say that this might be a place for us to investigate in follow-up surveys or other work.